modules/ROOT/pages/db-util.adoc

```diff
@@ -220,9 +91,9 @@ There are two different ways to define the structure and content of your databas
 The MUnit DB server is based on H2. Complex structures, index and join commands may not work properly.
 
 [[db-structure-sql]]
-=== Defining the DB Structure from an SQL File
+=== Defining the DB Structure From an SQL File
```

Comment:
"from" in titles is in lowercase, per the Chicago tab within Capitalize My Title (https://capitalizemytitle.com/).
Alas, the tool defaults to APA style rather than Chicago style.... 
(See screen cap I'll send in Slack confirmation of edit being done.)